lib/matplotlib/table.py

```diff
@@ -14,7 +14,7 @@
 You can add additional cells outside this range to have convenient
 ways of positioning more interesting grids.
 
-Author    : John Gill <jng@europe.renre.com>
```

Comment:
:sheep: Sorry, did not click through your user name to see your real name and connect that this is your own email address!

As this is you, do as you please.